src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -163,14 +157,18 @@ export default class PeopleAndRoles extends Mixins(EntityFilterMixin) {
   private showOrgPersonForm: boolean = false
   private activeIndex: number = -1
   private addEditInProgress: boolean = false
-  private currentOrgPerson: OrgPersonIF | null = null
+  private currentOrgPerson: OrgPersonIF = null
 
-  // enums for template
+  // Enums for template
   readonly CorpTypeCd = CorpTypeCd
   readonly RoleTypes = RoleTypes
   readonly IncorporatorTypes = IncorporatorTypes
   readonly NumWord = NumWord
 
+  private get orgPersonList (): OrgPersonIF[] {
+    return this.getAddPeopleAndRoleStep.orgPeople
+  }
```

Comment:
I created this local getter to extract orgPersonList, which is used a lot in this file, so I didn't have to change too much.